src/utils.js

```diff
@@ -103,11 +103,6 @@ export function sanitize(str) {
   return sanitizeName(str, { replacement: '-' });
 }
 
-export function hoursAsInteger(property, hours) {
-  if (Number.isInteger(hours)) return { [property]: hours };
-  return { [`${property}_in_minutes`]: Math.round(hours * 60) };
```

Comment:
This function alway read a bit weird to me and I think that the root of this bug might reside in line 108 but unable to confirm 100%.